GliaWidgetsTests/ViewModel/Chat/ChatMessagaEntryViewTests.swift

```diff
@@ -0,0 +1,30 @@
+@testable import GliaWidgets
+import XCTest
+
+class ChatMessageEntryViewTests: XCTestCase {
+    let messagageEntryView = ChatMessageEntryView.mock()
+
+    func test_text_view_height_with_more_than_max_lines() {
+        let exampleText = "This\nis a\nfive\nlines\ntext"
+        let maxTextLines = CGFloat(messagageEntryView.maxTextLines)
+        let oneLineHeight = Theme().chatStyle.messageEntry.messageFont.lineHeight
```

Comment:
I am not familiar enough with this logic, but does this getting the line height and then multiplying to get the text view height works well with other fonts that are not San Francisco?

Reply:
Yes, the lineHeight property of UIFont is designed to work with any font, not just San Francisco. When you use a different font, the lineHeight property will return a different value that is appropriate for that font. Different fonts can have different line heights even for the same point size, because the line height is part of the design of the font and is set by the font designer. This is why it's important to use the lineHeight property rather than hard-coding a specific line height, because it ensures that your layout respects the design of the font.

Reply:
Great information here :) thank you